### ISSUE TITLE
False positive with arrow filter =>

### ISSUE THREAD

**Issue by laurentmuller** [friendsoftwig/twigcs#120]
When used the `filter` like follow:

```Twig
{% set sizes = [34, 36, 38, 40, 42] %}
{{ sizes|filter(v => v > 38)|join(', ') }}
```
An error is reported:

`L.2   C.19 : ERROR    There should be 1 space(s) before ">".`

This is the same for the `map` and `reduce` filters:
```Twig
{% set people = [
    {first: "Bob", last: "Smith"},
    {first: "Alice", last: "Dupond"},
] %}
{{ people|map(p => "#{p.first} #{p.last}")|join(', ') }}
```
```Twig
{% set numbers = [1, 2, 3] %}
{{ numbers|reduce((carry, v) => carry + v) }}
```
 The check spacing must update to take account about the arrow.

**Comment by OwlyCode** [comment]
What version are you using? I reproduced the bug with `3.2.2` but not with the beta of the `4.0.0`.

**Comment by laurentmuller** [comment]
When I type:

`twigcs -V` the output is `Console Tool`.

I have installed with the command
```
composer global require friendsoftwig/twigcs
```
and the version is `3.2.2` when I look to the `composer.json`.

**Comment by laurentmuller** [comment]
When I install with:
```
composer global require friendsoftwig/twigcs:dev-master
```
The error disappears.

**Comment by OwlyCode** [comment]
I see, `3.2.2` does not support arrow functions. We had to refactor the tool in order to add proper support for it and this is only available on dev-master or `4.0.0-BETA3`.

If you've not created custom rulesets, I strongly recommend you use `4.0.0-BETA3`.

**Comment by laurentmuller** [comment]
Ok, I can live with the 4.0.0 version.
